java/lance-jni/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "lance-jni"
+version = "0.1.0"
```

Comment:
should this version stay consistent with the rust crate?

Reply:
Sure i can use the rust version. Still need to find a way to bump maven version i guess.